### DISCUSSION TITLE
Encoding chat

### DISCUSSION THREAD

**Discussion by squallp** [munafio/chatify#298] [General]
HI, 
first compliments for your excellent work. The whole project is really readable. 
I tried to encode messages and it is no big deal except for fetching live messages which you  do with .js , do you have a better idea than to customize your .js wildly? Or maybe a link to read because I'm not sure I understand laravel that well. I cannot see where you store messages in database, in messages model you are calling something UUID and I don't know how to access it.

**Comment by munafio** [comment]
@squallp to control message then you have to publish the controllers then modify MessagesController or even if needed modify the main class of Chatify which is ChatifyMessenger by extending it.. everything written in the official documentation.
